simulator/src/ux.js

```diff
@@ -695,3 +695,785 @@ async function postUserIssue(message) {
         }
     });
 }
+
+var isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
+
+if (isMobile) {
+    var head = document.getElementsByTagName('HEAD')[0];
+
+    // Create new link Element
+    var style = document.createElement('style');
+    style.textContent = `
+@media screen and (max-device-width: 1366px) {
```

Comment:
This looks hackish and updating css will be harder in this format. But since we don't have enough time, we can create another issue for cleaner implementation and deploy this PR for now